content/en/docs/tasks/administer-cluster/dns-custom-nameservers.md

```diff
@@ -59,15 +59,16 @@ inheriting DNS. Set it to a valid file path to specify a file other than
 
 ## CoreDNS
 
-CoreDNS is a general-purpose authoritative DNS server that can serve as cluster DNS, complying with the [dns specifications](https://github.com/kubernetes/dns/blob/master/docs/specification.md).
+CoreDNS is a general-purpose authoritative DNS server that can serve as cluster DNS,
+complying with the [DNS specifications](https://github.com/kubernetes/dns/blob/master/docs/specification.md).
 
 ### CoreDNS ConfigMap options
 
-CoreDNS is a DNS server that is modular and pluggable, and each plugin adds new functionality to CoreDNS.
-This can be configured by maintaining a [Corefile](https://coredns.io/2017/07/23/corefile-explained/), which is the CoreDNS
-configuration file. As a cluster administrator, you can modify the
-{{< glossary_tooltip text="ConfigMap" term_id="configmap" >}} for the CoreDNS Corefile to change how DNS service discovery
-behaves for that cluster.
+CoreDNS is a DNS server that is modular and pluggable, with plugins adding new functionalities.
+The CoreDNS server be configured by maintaining a [Corefile](https://coredns.io/2017/07/23/corefile-explained/),
```

Comment:
missing word
The CoreDNS server can be configured by maintaining a ....
OR
The CoreDNS server is configured by maintaining a ....